src/Unitverse.ExampleGenerator/Resources/AutomaticMockGeneration.txt

```diff
@@ -56,9 +56,14 @@ namespace Unitverse.Examples
         public async Task SampleAsyncMethod()
         {
             await _dummyService.AsyncMethod();
-            await _dummyService2.AsyncMethod("foo");
+            await SamplePrivateMethod();
```

Comment:
Could we add some extra samples here (like the more complex examples mentioned in the discussion for #190)? I think what you have written may actually capture lambda invocations, and it would be good to assert that too.